src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -9730,6 +9728,8 @@ DECLARE_API(GCWhere)
     return Status;
 }
 
+#ifndef FEATURE_PAL
+
 DECLARE_API(FindRoots)
 {
 #ifndef FEATURE_PAL
```

Comment:
I noticed you didn't include the CMakeFiles.txt change. Would you rather I get that in?